ts/features/itwallet/common/components/ItwEidInfoBottomSheetContent.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { View } from "react-native";
+import {
+  Divider,
+  HStack,
+  Icon,
+  IOStyles,
+  VStack,
+  H4,
+  Alert
+} from "@pagopa/io-app-design-system";
+import * as O from "fp-ts/lib/Option";
+import { constNull, pipe } from "fp-ts/lib/function";
+import I18n from "../../../../i18n";
+import { useIOSelector } from "../../../../store/hooks";
+import { itwCredentialsEidSelector } from "../../credentials/store/selectors";
+import IOMarkdown from "../../../../components/IOMarkdown";
+import { format } from "../../../../utils/dates";
+import { parseClaims, WellKnownClaim } from "../utils/itwClaimsUtils";
+import { StoredCredential } from "../utils/itwTypesUtils";
+import { ItwCredentialClaim } from "./ItwCredentialClaim";
+
+export const ItwEidInfoBottomSheetTitle = () => (
+  <HStack space={8} style={IOStyles.alignCenter}>
+    <Icon name="legalValue" color="blueIO-500" />
+    <H4>
+      {I18n.t("features.itWallet.presentation.bottomSheets.eidInfo.title")}
+    </H4>
+  </HStack>
+);
+
+export const ItwEidInfoBottomSheetContent = () => {
+  const eidOption = useIOSelector(itwCredentialsEidSelector);
+
+  const Content = ({ credential }: { credential: StoredCredential }) => {
+    const claims = parseClaims(credential.parsedCredential, {
+      exclude: [WellKnownClaim.unique_id, WellKnownClaim.content]
+    });
+
+    return (
+      <VStack space={24}>
+        <IOMarkdown
+          content={I18n.t(
+            "features.itWallet.presentation.bottomSheets.eidInfo.contentTop"
+          )}
+        />
+        <View>
+          {claims.map((claim, index) => (
+            <React.Fragment key={index}>
+              {index !== 0 && <Divider />}
+              <ItwCredentialClaim claim={claim} isPreview={true} />
+            </React.Fragment>
+          ))}
+        </View>
+        {credential.jwt.issuedAt && (
+          <Alert
+            variant="success"
+            content={I18n.t(
+              "features.itWallet.presentation.bottomSheets.eidInfo.alert.valid",
+              { issuanceDate: format(credential.jwt.issuedAt, "DD-MM-YYYY") }
+            )}
+          />
+        )}
+        <IOMarkdown
+          content={I18n.t(
+            "features.itWallet.presentation.bottomSheets.eidInfo.contentBottom"
+          )}
+        />
```

Comment:
Looking at the Figma, these two components should be inverted. Is it correct?

Reply:
I followed the [Approved Only](https://www.figma.com/design/pVD9XCba7crkg5yOIdJCXl/%5BIT-Wallet%5D---%E2%9C%85-Approved-only-%E2%9C%85?node-id=4344-70660&node-type=frame&t=u1GlDDuypaqnzIuT-0) Figma. The alert is before the last block of text.